docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc

```diff
@@ -7,6 +7,123 @@ The following lists prebuilt rule updates per release. Only rules with
 significant modifications to their query or scope are listed. For detailed
 information about a rule's changes, see the rule's description page.
 
+[float]
+=== 7.12.0
+
+
+This prebuilt rule has been deprecated:
+
+* setgid-bit-set-via-chmod
```

Comment:
```suggestion
* setuid-setgid-bit-set-via-chmod
* execution-via-regsvcs-regasm
```

Reply:
Should be ok as is: https://github.com/elastic/security-docs/pull/566#discussion_r603475100